dolphinscheduler-task-plugin/dolphinscheduler-task-sagemaker/src/main/java/org/apache/dolphinscheduler/plugin/task/sagemaker/PipelineUtils.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.task.sagemaker;
+
+import org.apache.dolphinscheduler.common.thread.ThreadUtils;
+import org.apache.dolphinscheduler.plugin.task.api.TaskConstants;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.amazonaws.services.sagemaker.AmazonSageMaker;
+import com.amazonaws.services.sagemaker.model.DescribePipelineExecutionRequest;
+import com.amazonaws.services.sagemaker.model.DescribePipelineExecutionResult;
+import com.amazonaws.services.sagemaker.model.ListPipelineExecutionStepsRequest;
+import com.amazonaws.services.sagemaker.model.ListPipelineExecutionStepsResult;
+import com.amazonaws.services.sagemaker.model.PipelineExecutionStep;
+import com.amazonaws.services.sagemaker.model.StartPipelineExecutionRequest;
+import com.amazonaws.services.sagemaker.model.StartPipelineExecutionResult;
+import com.amazonaws.services.sagemaker.model.StopPipelineExecutionRequest;
+import com.amazonaws.services.sagemaker.model.StopPipelineExecutionResult;
+
+public class PipelineUtils {
```

Comment:
Maybe we could add some basic UTs for this class? Currently test coverage for this PR is a little bit low.

Reply:
How can I see the UTs coverage of my PR alone

Reply:
I'm not sure about `sonar`, could u plz take a look? Thx~@caishunfeng
BTW, @jieguangzhou you may use your IDE to see the coverage by running your UT in `run with coverage` mode. Actually, you could estimate the coverage by just looking into the code...

Reply:
Since it is a PR for a new task plugin, I don't have strong opinion about the UT coverage, just a suggestion here.

Reply:
Thanks for your advice. I just went to test it with the IDE. The results are as follows

<img width="854" alt="image" src="https://user-images.githubusercontent.com/31528124/180140873-5da485a2-af2d-4a05-a9a7-03fc58b3a320.png">